src/tools/structures/traitement-coselec.ts

```diff
@@ -88,7 +88,7 @@ execute(__filename, async ({ app, logger, exit }) => {
       },
     },
   };
-  if (structure.statut === 'CREEE') {
+  if (Number(options.quota) > 0 && structure.statut !== 'VALIDATION_COSELEC') {
```

Comment:
|| statut CREEE

Reply:
yes le CREEE est inclus dans la condition => ` !== 'VALIDATION_COSELEC'`

Reply:
non pour les cas qui étaient en ABANDON ou ANNULEE par exemple

Reply:
ah oui oula

Reply:
heureusement que c'est le WE